assets/variant-list.css

```diff
@@ -367,4 +422,130 @@ variant-list-remove-button .icon-remove {
   .variant-list__table thead th:first-child {
     width: 37%;
   }
+
+  .variant-list-buttons {
+    display: flex;
+    align-self: flex-start;
+    width: 37%;
+  }
+
+  variant-list-remove-all-button {
+    margin-left: 0.9rem;
+  }
+}
+
+.variant-list__total-items {
+  display: flex;
+}
+
+.variant-list__total-items h3,
+.totals__subtotal-value {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.variant-list__total-items p,
+.totals__subtotal {
+  margin-top: 0.2rem;
+  opacity: 0.75;
+  margin-bottom: 1.2rem;
+}
+
+.variant-list__total-items h3,
+.totals__subtotal-value {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.variant-list__total-items p,
+.totals__subtotal {
+  margin-top: 0.2rem;
+  opacity: 0.75;
+  margin-bottom: 1.2rem;
+}
+
+.variant-list-total__info {
+  display: flex;
+}
+
+.variant-list-total {
+  padding-top: 2rem;
+  border-top: 0.1rem solid rgba(var(--color-foreground), .08);
+}
+
+.totals__product-total {
+  text-align: center;
+}
+
+@media screen and (min-width: 750px) {
+  .variant-list__total-items {
+    padding-left: calc(5rem / var(--font-body-scale) + var(--inputs-border-width) * 2);
+  }
+
+  .totals__product-total {
+    text-align: right;
+  }
+
+  .variant-list__total-items {
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .totals {
+    flex-grow: 1;
+    text-align: right;
+  }
+
+  .variantlist-total {
+    position: sticky;
+    bottom: 0;
+    z-index: 1;
+    background-color: rgb(var(--color-background));
+  }
+
+  .variant-list__total-items {
+    padding-left: calc(5rem / var(--font-body-scale) + var(--inputs-border-width) * 2);
+  }
+
+  .totals__product-total {
+    text-align: right;
+  }
+
+  .variant-list__total-items {
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .totals {
+    flex-grow: 1;
+    text-align: right;
+  }
+}
+
+
+@media screen and (max-width: 750px) {
+  .totals__product-total {
+    display: flex;
+    padding-top: 2rem;
+    border-top: 0.1rem solid rgba(var(--color-foreground), .08);
```

Comment:
this border line doesn't match table line on mobile 🤔
https://screenshot.click/13-45-t42kx-ut04v.mp4

Reply:
Good catch!